### PR TITLE
Added blog post about SIG OBS TL.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: web/content/posts/sig-obs-tl.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+It would be awesome to maintain a good relationship with such projects, allow even better integration with them, communicate more, and help each other!
+ 
+> **Action Item**: If you feel that some projects or initiatives are extremely useful, but missing in Observability the CNCF portfolio, or something we can learn [reach us as well!](#how-you-can-reach-us)
+
+## Summary
+
+![View from Pyrgos village on Santorini island, Greece](/images/blog/sig-obs-tl/santorini2.jpg)
+
+I hope with this blog post, you learned a bit what this magic `SIG` concept is all about and how `YOU` can help within the area you are
```
Comment (kakkoyun) [inline web/content/posts/sig-obs-tl.md:206]:
```suggestion
I hope with this blog post, you have learned a bit what this magical `SIG` concept is all about. And how `YOU` can help within the domain you are
```

File: web/content/posts/sig-obs-tl.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+passionated the most about as well!
```
Comment (kakkoyun) [inline web/content/posts/sig-obs-tl.md:207]:
```suggestion
passionated about!
```

File: web/content/posts/sig-obs-tl.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+Thanks [all who voted for me](https://lists.cncf.io/g/cncf-toc/topic/76717430#5272) and particularly thanks to [Richi](https://twitter.com/twitchih)
```
Comment (kakkoyun) [inline web/content/posts/sig-obs-tl.md:209]:
```suggestion
Thanks to [all who voted for me](https://lists.cncf.io/g/cncf-toc/topic/76717430#5272) and especially thanks to [Richi](https://twitter.com/twitchih)
```

**Review by kakkoyun — COMMENTED** [review]
🥇

**Comment by bwplotka** [comment]
Massive thanks for review @kakkoyun :heart:

**Pull request merged**